tests/Base.php

```diff
@@ -10,8 +10,11 @@
 
 class Base extends TestCase
 {
+    private string $runtime = '';
+
     public function setUp(): void
     {
+        $this->runtime = \explode('-', \getenv('RUNTIME'), 2)[0] ?? '';
```

Comment:
This wont work well for `python-ml-3.11`, for example. We need to split by `-`, remove last from array, and join back with `-`. With that logic we get `python-ml` proplerly.

Reply:
Makes sense. Will check.

Reply:
Done.